package.json

```diff
@@ -5,5 +5,8 @@
   "main": "index.js",
   "repository": "https://github.com/coleturner/pettier.git",
   "author": "Cole Turner <turner.cole@gmail.com>",
-  "license": "MIT"
+  "license": "MIT",
+  "devDependencies": {
+    "prettier": "^1.13.5"
+  }
 }
```

Comment:
Would you mind adding a precommit hook for this?

Reply:
yea sure